public/app.css

```diff
@@ -0,0 +1,27 @@
+@font-face {
+  font-family: Bariol;
+  font-weight: bold;
+  src: url('./fonts/Bariol_Bold.otf') format('opentype');
```

Comment:
This `app.css` file is just the sandbox in which we load the viz, so it mimics the wrapping page, there won't be any duplication.